### PR TITLE
Rename oomd to oomagent

### LINKED ISSUE #645: Rename oomctl to oomcli and oomd to oomagent
oomctl is taken by systemd that is built into Linux

### PR DESCRIPTION
**Pull request by wfxr**
Close #645

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,7 @@ test:
 .PHONY: integration-test
 integration-test: codegen
 	$(MAKE) -C oomcli integration-test
-	$(MAKE) -C oomd    integration-test
+	$(MAKE) -C oomagent    integration-test
```
Comment (YikSanChan) [inline Makefile:50]: less spaces

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by jinghancc — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (wfxr) [reply Makefile]: fixed

**Pull request merged**